lib/paneron/register/data_set_metadata/contact.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "lutaml/model"
+
+module Paneron
+  module Register
+    class DataSetMetadata < Lutaml::Model::Serializable
+      class Contact < Lutaml::Model::Serializable
+        attribute :label, Lutaml::Model::Type::String, default: -> { "email" }
+        attribute :value, Lutaml::Model::Type::String
+        attribute :notes, Lutaml::Model::Type::String # NOTE: There is no { optional: true } # Use render_nil: true for all other attributes in the render block instead
```

Comment:
All attributes are optional unless you set a required cardinality. Or is there something else you need? `render_nil` is really only for compatibility reasons...

Reply:
Yes, with the more verbose form, I was also hoping my LSP could point me to the definition...